galette/templates/default/pages/contributions_list.html.twig

```diff
@@ -299,12 +299,7 @@
                 {% else %}
                     {{ ordre + 1 + (filters.current_page - 1) * numrows }}
                 {% endif %}
-                {% if (login.isAdmin() or login.isStaff()) or mode != 'ajax' %}
-                    <span class="displaynone">
-                        <a href="{{ url_for("editContribution", {"type": ctype, "id": contribution.id}) }}">
-                            {{ _T("Contribution %id")|replace({"%id": contribution.id}) }}
-                        </a>
-                    </span>
```

Comment:
What is the purpose of this hidden link ?
There is a similar one on transactions list.

Reply:
Before being a `displaynone`; class was `row-title`. That was used in mobile view.